docs/qa/CheckLists/FavoritePage_tags.md

```diff
@@ -0,0 +1,68 @@
+| 36 | Buttons on bottom menu are 1) Export 2) Button move to tag 3) Copy to tag 4) delete                                                                                                                                                                      |             |                                                                                    |
+| 37 | When pressing "Export" button 19 bottom sheet opens up                                                                                                                                                                                                   |             |                                                                                    |
+| 38 | When pressing "Move to tag" button 18 bottom sheet opens up                                                                                                                                                                                              |             |                                                                                    |
+| 39 | When pressing "Copy to tag button 17 bottom sheet opens up                                                                                                                                                                                               |             |                                                                                    |
+| 40 | When pressing "delete" button 16 bottom sheet opens up                                                                                                                                                                                                   |             |                                                                                    |
+| 41 | The only way to stop this mode is to press "back" button at the upper left                                                                                                                                                                               |             |                                                                                    |
+| 42 | From top to bottom 1) Selected "x" where x is a number of selected tags 2) Select all tags button, 3) statistics on how many tags and words are presented 4) Tags list                                                                                   |             |                                                                                    |
+| 43 | Pressing select all tags button when it is not activated will select all tags                                                                                                                                                                            |             | What happens if the user manually chooses all tags? Will it activate itself?       |
```

Comment:
User has two options here:
1. Select all tags with the 'Select all tags' button
2. Manually select all tags one-by one
They both work the same.